src/amo/components/Addon/index.js

```diff
@@ -488,7 +488,8 @@ export class AddonBase extends React.Component {
             {addon && addonType === ADDON_TYPE_THEME && (
               <AddonsByAuthorsCard
                 addonType={addonType}
-                authorNames={addon.authors.map((author) => author.username)}
+                authorDisplayName={addon.authors[0].name}
```

Comment:
In `AddonsByAuthorsCard`, if there is more than one author, it [will always display a message instead of an author name](https://github.com/mozilla/addons-frontend/blob/master/src/amo/components/AddonsByAuthorsCard/index.js#L112-L163), e.g., `More dictionaries by these translators` and `More extensions by these developers`. So if there is more than one author we don't actually care what's passed into `authorDisplayName` and if there is only one author we want that name, and that will be `addon.authors[0].name`. 😄